src/callbacks_step/stepsize_dg1d.jl

```diff
@@ -16,10 +16,12 @@ function max_dt(u, t, mesh::TreeMesh{1},
         for i in eachnode(dg)
             u_node = get_node_vars(u, equations, dg, i, element)
             lambda1, = max_abs_speeds(u_node, equations)
-            max_lambda1 = max(max_lambda1, lambda1)
+            max_lambda1 = Base.max(max_lambda1, lambda1)
         end
         inv_jacobian = cache.elements.inverse_jacobian[element]
-        max_scaled_speed = max(max_scaled_speed, inv_jacobian * max_lambda1)
+        # Use `Base.max` to prevent silent failures, as `max` from `@fastmath` doesn't propagate
+        # `NaN`s properly. See https://github.com/trixi-framework/Trixi.jl/pull/2445#discussion_r2336812323
```

Comment:
Does it make sense to put the comment to the first usage of `Base.max`?

Reply:
I put it to the least indented line, but I don't have a strong opinion on that.

Reply:
Hm yeah I guess that is also a reason